dace/runtime/include/dace/cuda/dynmap.cuh

```diff
@@ -25,7 +25,11 @@
 #ifndef __DACE_DYNMAP_CUH
 #define __DACE_DYNMAP_CUH
 
+#ifdef __CUDACC__
 #include <cooperative_groups.h>
+/*#elif defined(__HIPCC__)
+#include <hip/hip_cooperative_groups.h>
```

Comment:
Commented out?

Reply:
For now, cooperative groups in HIP do not implement the same cooperative groups in CUDA. Added a comment.